data/tmpl/values/content5.html

```diff
@@ -66,6 +66,7 @@ <h2 class="token-awarded hidden" aria-hidden="true">Here’s your badge!</h2>
       </footer>
     </section>
     <script src="/data/js/lib/token-manager.js"></script>
+    <script src="/data/js/lib/common.js"></script>
```

Comment:
Nit: I would suggest loading `common.js` before loading `token-manager.js` here. It is more likely that `token-manager.js` will depend on something in `common.js` that the other way around. Thoughts?